.github/workflows/load-testing.yml

```diff
@@ -0,0 +1,147 @@
+name: Baseline Load Tests
+
+permissions: {}
+
+on:
+  pull_request:
+    branches:
+      - "main"
+      - "release*"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  prepare-images:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - name: Setup caches
+        uses: ./.github/actions/setup-caches
+        timeout-minutes: 5
+        continue-on-error: true
+        with:
+          build-cache-key: build-images
+      - name: Setup build env
+        uses: ./.github/actions/setup-build-env
+        timeout-minutes: 10
+        with:
+          free-disk-space: false
+      - name: ko build
+        shell: bash
+        run: |
+          set -e
+          VERSION=${{ github.ref_name }} make docker-save-image-all
+      - name: upload images archive
+        uses: actions/upload-artifact@a8a3f3ad30e3422c9c7b888a15615d19a852ae32 # v3.1.3
+        with:
+          name: kyverno.tar
+          path: kyverno.tar
+          retention-days: 1
+          if-no-files-found: error
+
+  load-test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        kyverno-config:
+          - name: default
+            values:
+              - default-with-profiling
+          - name: standard
+            values:
+              - standard-with-profiling
+        test:
+          - kyverno-pss
+          - kyverno-mutate
```

Comment:
Where can I find these policies?

Reply:
They are in the load testing repo. The first one installs the PSS policiewhile the second one installs its own policy.

Reply:
Can you link to it?

Reply:
https://github.com/kyverno/load-testing/blob/main/k6%2Ftests%2Fkyverno-mutate.js

https://github.com/kyverno/load-testing/blob/main/k6%2Ftests%2Fkyverno-pss.js